src/components/CheckBox.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { View, Text, Image, StyleSheet } from 'react-native';
+import { RectButton } from 'react-native-gesture-handler';
+
+import { icons, theme } from '../constants';
+
+const { FONTS, COLORS } = theme;
+const BOX_SIZE = 11;
+
+const CheckBox = ({ label }) => {
+    const [ isChecked, setIsChecked ] = useState(false);
+
+    return (
+        <View style={styles.checkBoxContainer}>
+            <RectButton 
+                style={[styles.box, 
+                    { 
+                        backgroundColor: isChecked 
+                        ? COLORS.primary
+                        : COLORS.white,
+                        borderColor: isChecked
+                        ? COLORS.primary
+                        : COLORS.black
+                    }]}
+                onPress={() => setIsChecked(!isChecked)}
```

Comment:
I don't know if I would need to install a package :c, but when I run your project for the first time, the checkboxes appear blank and I can't see if they are really there or not, although when I click on the space it gives me the style of "_IsChecked_". If you have that color, WDYT about changing it to one that is easier to perceive? Otherwise, if it's just an error of my execution then there is no problem  😊 

**Screenshot when I run it:**

![Screenshot_1618261816](https://user-images.githubusercontent.com/69128578/114463098-98301b80-9ba9-11eb-9fad-2f2a5132bf1a.png)

**Screenshot when I click on the blank spaces:**

![Screenshot_1618261881](https://user-images.githubusercontent.com/69128578/114463213-c01f7f00-9ba9-11eb-85f4-b552c084b4b0.png)

Reply:
You are right, @JacquelineFM , thanks for the comment, we have already changed the color of the CheckBox to make it easier to perceive when it is not selected.

Reply:
Excellent! I'll check it right away 😊